Shell/Options.cpp

```diff
@@ -3215,7 +3222,6 @@ void Options::readOptionsString(vstring optionsString,bool assign)
                 USER_ERROR("value "+value+" for option "+ param +" not known");
                 break;
               case IgnoreMissing::WARN:
-                env.beginOutput();
```

Comment:
I assume deliberate.

Reply:
Yes, absolutely! Pet peeve.

Reply:
If I recall, one of the things beginOutput does is take a lock on output stream so that multiple processes can't write at the same time... which is useful/important.

Reply:
This one is actually (mildly) interesting.

`beginOutput() / endOutput()` do some locking but only if a `pipe` has been set as the output sink. Which is done by `setPipeOutput` only ever called by `tTwoVampires.ccp` close to a very historic line 

`runChild(units, "dis+10_32_nwc=2.0:sac=on:spl=backtracking_20"); //start proving`

we also have `Environment::setPriorityOutput` never called at all. So for all practical purposes `Environment::out()` is the same as `cout` and `beginOutput() / endOutput()` never does anything useful. (Unless we want to go back to pipes and priorities, of course.)